downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/MapOldStableIds.java

```diff
@@ -0,0 +1,127 @@
+package org.reactome.release.downloadDirectory;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MapOldStableIds {
+	private static final Logger logger = LogManager.getLogger();
+	private static final String filename = "reactome_stable_ids.txt";
+	
+	public static void execute(String username, String password, String host, String releaseNumber) throws Exception {
+		
+		logger.info("Running MapOldStableIds step");
+		// Need to use mysql driver to access stable_identifiers db
+		logger.info("Connecting to stable_identifers db...");
+		Class.forName("com.mysql.jdbc.Driver");
+		Connection connect = DriverManager.getConnection("jdbc:mysql://" + host + "/stable_identifiers?" + "user=" + username + "&password=" + password);
```

Comment:
See the comment RE: Connection objects, which I left on `FetchTestReactomeOntologyFiles`.

Reply:
The reasoning is slightly different than in FetchTestReactomeOntologyfiles -- This is accessing the stable_identifiers database, and this Database doesn't work the the Reactome DBAdaptor (stable_identifiers doesn't contain a Data Model table, which is needed)

Reply:
Aha! That makes sense. :+1: